docs/Image.md

```diff
@@ -27,6 +27,10 @@ When your entire codebase respects this convention, you're able to do interestin
 
 *This process is currently being improved, a much better workflow will be available shortly.*
 
+> **NOTE**: PNG images are required when loading with `require('image!my-icon')`
+>
+> At this time, only PNG images are supported in iOS. There is an [issue](https://github.com/facebook/react-native/issues/646) that is currently addressing this bug. In the meantime a quick fix is to rename your files to image.png or to use the `isStatic` flag like: `source={{ uri: 'image', isStatic: true }}`.
```

Comment:
nit: please be consistent with your examples, in this case change `uri` to `my-icon`